examples/README.md

```diff
@@ -50,6 +50,39 @@ kubectl replace -f ./csi-config-map-sample.yaml
 Storage class and snapshot class, using `<cluster-id>` as the value for the
 option `clusterID`, can now be created on the cluster.
 
+## Running CephCSI with pod networking
+
+The current problem with Pod Networking, is when a CephFS/RBD volume is mounted
+in a pod using Ceph CSI and then the CSI CephFS/RBD plugin is restarted or
+terminated (e.g. by restarting or deleting its DaemonSet), all operations on
+the volume become blocked, even after restarting the CSI pods.
+
+The only workaround is to restart the node where the Ceph CSI plugin pod was
+restarted.  This can be mitigated by running the `rbd map`/`mount -t` commands
+in a different namespace which does not get deleted and recreated often.
+
+If someone wants to run the CephCSI with the pod networking they can still do
+by setting the `netNamespaceFilePath`. If this path is set CephCSI will execute
+the `rbd map`/`mount -t` commands by entering the network namespace specified by
+`netNamespaceFilePath` with the nsenter.
```

Comment:
> ... with the `nsenter` command.

Or just

> ... with `nsenter`.

Might be good to reference `man 7 namespaces` and/or `man 7 network_namespaces` for additional information.

Reply:
added link to man page